sysutils/mise/Portfile

```
@@ -19,15 +19,17 @@ long_description    mise is a front-end to your dev environment. \
 categories          sysutils
 license             MIT
 
-maintainers         {outlook.com:gjq.uoiai @MisLink} openmaintainer
+maintainers         {outlook.com:gjq.uoiai @MisLink} \
+                    {gmail.com:herby.gillot @herbygillot} \
```

I am also happy to co-maintain mise, as I use it for version management over asdf and vfox.

```suggestion
                    {gmail.com:herby.gillot @herbygillot} \
                    {macports.halostatue.ca:austin @halostatue} \
```

---

Sounds good 👍🏾